### DISCUSSION TITLE
How to completely uninstall this?

### DISCUSSION THREAD

**Discussion by Jaksonas** [ElPumpo/TinyNvidiaUpdateChecker#208] [Q&A] [answered]
Hello, how do I completely uninstall / stop this script from running?
However, problem is I do not remember how I installed it, so it would be great if you could provide all the ways to uninstall this. Thanks

**Comment by HawaiiBeach** [comment, marked as answer]
Hi. The uninstall is opposite of install. There's no official "install" as TNUC is a portable app.
